src/components/ContentSidebar/DetailsSidebar.scss

```diff
@@ -8,3 +8,11 @@
         margin-bottom: 0;
     }
 }
+
+.bcs-details-content {
+    margin: 0 20px 20px 30px;
+}
+
+.bcs-details-content:empty {
```

Comment:
Just double check if IE11 is fine with this.

Reply:
Since IE9 https://developer.mozilla.org/en-US/docs/Web/CSS/:empty